Cargo.toml

```diff
@@ -22,7 +22,15 @@ futures = "0.3.26"
 pythonize = "0.20.0"
 serde = "1.0"
 serde_json = "1.0.91"
+# Lindera
+lindera-core = { version = "0.27.2", optional = true }
+lindera-dictionary = { version = "0.27.2", optional = true }
+lindera-tantivy = { version = "0.27.1", optional = true, features = ["ipadic"] }
```

Comment:
Are there different features that will be required to support?

Reply:
Since we probably do not want to expose all of Lindera's features as features of tantivy-py, I think we should go for a maximum set here.

Reply:
Done